### PR TITLE
switched to a date utility package included with core python

### PR DESCRIPTION
**Pull request by brentpicasso**

### CODE REVIEW

File: utils.py [hunk 1]
```diff
@@ -32,7 +35,10 @@ def boundary(value, minvalue, maxvalue):
     '''Limit a value between a minvalue and maxvalue'''
     return min(max(value, minvalue), maxvalue)
 
-
+def time_to_epoch(timestamp):
+    epoch = int(calendar.timegm(datetime.strptime(timestamp, "%Y-%m-%dT%H:%M:%SZ").timetuple()))
```
Comment (stieg) [inline utils.py]: why even define epoch here?  Just make it a one liner or refactor the code a bit so its easier to read.

File: utils.py [hunk 2]
```diff
@@ -9,12 +9,15 @@
            'is_color_transparent', 'boundary', 'dist',
            'deprecated', 'SafeList',
            'interpolate', 'OrderedDict', 'kvFind', 'kvFindClass', 'kvPrintAttr',
-           'breadth_first', 'walk_tree', 'filter_tree', 'kvquery', 'pct_h', 'pct_w')
+           'breadth_first', 'walk_tree', 'filter_tree', 'kvquery', 'pct_h', 'pct_w', 'time_to_epoch')
 
+import time
```
Comment (stieg) [inline utils.py]: Do we organize imports alphabetically?  Also why some use of `from X import Y` syntax as opposed to being consistent?

File: autosportlabs/racecapture/tracks/trackmanager.py [hunk 3]
```diff
@@ -4,7 +4,6 @@
 import errno
 import string
 import logging
-from dateutil.parser import *
```
Comment (stieg) [inline autosportlabs/racecapture/tracks/trackmanager.py]: Glad this is dying.

File: utils.py (hunk 1 above)
Comment (brentpicasso) [inline utils.py]: It was a habit from C / firmware code so it's easier to interactively debug. Don't do that much in python, it seems?

File: utils.py (hunk 2 above)
Comment (brentpicasso) [inline utils.py]: we don't ; it's semi arbitrarily organize it from core / high level packages to local imports, in that top to bottom order.

IDK on the other questions. seems some learning on importing python packages is in order.

File: utils.py (hunk 1 above)
Comment (brentpicasso) [inline utils.py]: Fixed / streamlined return.

**Pull request merged**